lib/native_images.dart

```diff
@@ -1,68 +0,0 @@
-import 'dart:async';
-import 'dart:typed_data';
-import 'dart:ui' as ui;
-
-import 'package:flutter/rendering.dart';
-import 'package:flutter/services.dart';
-import 'package:flutter/painting.dart';
-import 'package:flutter/foundation.dart'
-    show SynchronousFuture, describeIdentity;
-
-// Performs exactly like a MemoryImage but instead of taking in bytes it takes
-// in a future that represents bytes.
-class FutureMemoryImage extends ImageProvider<FutureMemoryImage> {
```

Comment:
Seems like you meant to have this in the flutter/flutter repo :D

Reply:
You think it should be there?  I didn't want to start pushing that rock up that hill.